examples/pojo-jar/README.md

```diff
@@ -0,0 +1,48 @@
+# camel-k-jar
+This repository takes the example from here https://camel.apache.org/manual/rest-dsl.html#_openapi_swagger_api
+and here https://github.com/apache/camel-examples/tree/camel-3.11.x/examples/openapi-cdi and make it work with Camel K.
+
+Target of this setup was to make the local development experience better,
+by providing a way to use not only inner classes in route file (`OpenApiRoute.java`) and avoid using
+Maven or jitpack, which add major round trips to the development cycle.
+
+## Solution
+### Build a JAR
+First you need to build a jar file of your source code. For example by 
```

Comment:
Would it be better to point to the POM and Java file, and provide the command to run to build the POJO archive?

Reply:
removed the xml and added mvn install command